src/Silverback.Integration/Messaging/Messages/DefaultMessageHeaders.cs

```diff
@@ -91,6 +91,14 @@ public static class DefaultMessageHeaders
         /// </summary>
         public const string ContentType = "content-type";
 
+        /// <summary>
+        ///     The encryption Key Identifier.
+        /// </summary>
+        /// <remarks>
+        ///     The header is required for the Key Rotation Feature.
```

Comment:
In general, the summaries must be more explanatory. I don't think that a random library user would immediately understand what we mean with _"key rotation feature"_.

Reply:
I adjusted the summary.